src/App.tsx

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
+import todos from './api/todos';
+import users from './api/users';
+import { TodoList } from './components/TodoList';
+
 import './App.scss';
 
-// import todos from './api/todos';
-// import users from './api/users';
+const preparedTodos = todos.map((todo) => {
+  const user = users.find(({ id }) => id === todo.userId);
+
+  return {
+    ...todo,
+    user,
+  };
+});
```

Comment:
можна скоротити код, якщо з map зразу повертати об'єкт

```suggestion
const preparedTodos = todos.map(todo => ({
  ...todos,
  user: users.find(({ id }) => id === todo.userId),
}));
```